app/lib/katello/resources/candlepin.rb

```diff
@@ -157,8 +157,8 @@ def checkin(uuid, checkin_date)
             self.put(path(uuid), {:lastCheckin => checkin_date}.to_json, self.default_headers).body
           end
 
-          def available_pools(owner_label, uuid, listall = false)
-            url = Pool.path(nil, owner_label) + "?consumer=#{uuid}&listall=#{listall}"
+          def available_pools(owner_label, uuid, listall = false, add_future = true)
```

Comment:
why use a default argument if it looks like only true is being used? My opinion would be to hardcode `add_future=true` in the URL and it can be updated to an argument in the future if need be.